### ISSUE TITLE
Minor: Icon Missing

### ISSUE THREAD

**Issue by jiiins** [Rudd-O/homeassistant-meteoswiss#25]
It looks like the integration's icon is missing, both in HACS and locally after installation.

**Comment by mastameista** [comment]
+1 same here :)

**Comment by Rudd-O** [comment]
I'd like that too, but it doesn't look like I control this — I think this is controlled by HACS and I tried the process, only to be rudely told no (this is a very small summary).

If anyone knows how to add the integration icon, without having to go through HACS, I would gladly do so.

**Comment by braetkuegeli** [comment]
According this [blog post](https://developers.home-assistant.io/blog/2020/05/08/logos-custom-integrations/) it must done via brands:

If this icons are ok for you, I would create a pull request:
https://github.com/home-assistant/brands/compare/master...braetkuegeli:brands:master

**Comment by Rudd-O** [comment]
@braetkuegeli yes please.  Thank you for creating the PR there.  I don't have access to create PRs in that repo.  When that integration is merged in the brands repo, icon will just work.

**Comment by Rudd-O** [comment]
Leaving this open until such a PR is merged.

**Comment by Superkikim** [comment]
I've just looked at the pull request. And frenck is complaining about the icon being from the app. 

He did exactly the same for me with my maxsmart integration. I couldn't sort it out (and took it quite personally I must say).

**Comment by Rudd-O** [comment]
Without too many details, I've gotten perfectly working PRs rejected from HA before.  I understand your frustration.

**Comment by Rudd-O** [comment]
This is resolved!

**Comment by Rudd-O** [comment]
Thank you @braetkuegeli .
